pom.xml

```diff
@@ -231,7 +231,13 @@
     <dependency>
       <groupId>pentaho</groupId>
       <artifactId>guava-cache-provider</artifactId>
-      <version>7.1-SNAPSHOT</version>
+      <version>${project.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.pentaho</groupId>
+      <artifactId>pdi-engine-api</artifactId>
```

Comment:
Hmmm.  There's nothing in your change that would introduce that dependency.  How would tests before w/o, and now require it?